ducktape/cluster/json.py

```diff
@@ -55,13 +54,12 @@ def request(self, nslots):
         result = []
         for i in range(nslots):
             node = self.available_nodes.popleft()
-            result.append(ClusterSlot(self, node, slot_id=self.id_source))
-            self.in_use_nodes.add(self.id_source)
-            self.id_source += 1
+            result.append(node)
```

Comment:
This changes the type of output for this method from list of ClusterSlots to list of RemoteAccounts, but there's not a corresponding change to LocalhostCluster.

Also, the ClusterSlot import is no longer used and can be removed.